src/lib/store-cash/store-cash-fpti.js

```diff
@@ -0,0 +1,149 @@
+/* @flow */
+import { getMerchantID } from '@paypal/sdk-client/src';
+
```

Comment:
any chance this file can be generic, not just to store cash. seems like it's a `deprecated pp-shopping fpti event`
Than way it can be used, if needed, to fire the alternative fpti event